sample_data_config/web-api/web-api-data-pipeline.config.yaml

```diff
@@ -20,21 +20,71 @@ webApi:
         - items
   #elifescience profile api
   - dataset: '{ENV}'
-    table: atest_profile_api
+    table: test_people_api
     dataUrl:
-      urlExcludingConfigurableParameters: https://api.elifesciences.org/profiles?
+      urlExcludingConfigurableParameters: https://api.elifesciences.org/people?
       configurableParameters:
         pageSizeParameterName: per-page
         defaultPageSize: 100
         pageParameterName: page
     response:
       itemsKeyFromResponseRoot:
         - items
+  # civi export
+  - dataset: '{ENV}'
+    table: civi_data_export
+    stateFile:
+      bucketName: '{ENV}-elife-data-pipeline'
+      objectName: 'airflow-config/generic-web-api/{ENV}-civi-export-processing-state.json'
+    urlSourceType:
+      name: 'civi'
+      sourceTypeSpecificValues:
+        fieldsToReturn:
+          - external_identifier
+          - tag
+          - group
+          - contact_type
+          - contact_sub_type
+          - last_name
+          - first_name
+          - middle_name
+          - email
+          - source
+          - user_unique_id
+          - organization_name
+          - phone
+          - city
+          - modified_date
```

Comment:
Just making a note that `modified_date` is returned even if not specified.

Reply:
i have a different opinon about this though, one should specify it if one think it is useful, besides, i am not entirely certain this field is present for all entity types in civi. I'll implement your suggestion though

Reply:
This was just a comment / observation on my side.